src/App.js

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import ListView from './components/ListView';
+import AddView from './components/AddView';
+import Home from './components/Home';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+// https://www.newline.co/@andreeamaco/how-to-handle-navigation-in-your-app-with-react-router-link--088f82d3
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <Switch>
+        <Route exact path="/list-view">
+          <ListView />
+        </Route>
+        <Route exact path="/add-view">
+          <AddView />
+        </Route>
+        <Route exact path="/">
```

Comment:
Hi Luka, I've removed the `exact` params, as you suggested

Reply:
It looks like you've removed `exact` from all of them, which is 👍 . Here's the behavior:

If it matches with the list or add paths, it will render those components. If it doesn't, the `/` path not having an `exact` param on it will act as a catch-all for any paths not defined.

Say you wanted to show a `404 Not Found` page, then you could create another `/` path for it, and have the home component render only on `exact` matches.

It would end up looking something like:
```jsx
<Route exact path="/">
    <Home />
</Route>
<Route path="/">
    <NotFound />
</Route>
```

React may have a better way to do this, I'm just extrapolating here. Also, this is just for informational use, definitely out of scope.